spec/views/users/users_show_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'User Page Show', type: :feature do
+  before :each do
+    @user = User.create(
+      name: 'Tom',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'I love cats',
+      posts_counter: 3
+    )
+    @post1 = Post.new(
+      title: 'First Post',
+      text: 'here we go again',
+      comments_counter: 2,
+      likes_counter: 3,
+      author_id: @user.id
+    )
+    @post2 = Post.new(
+      title: 'Second Post',
+      text: 'This is my second post',
+      comments_counter: 2,
+      likes_counter: 3,
+      author_id: @user.id
+    )
+    @post3 = Post.new(
+      title: 'Third Post',
+      text: 'This is my third post',
+      comments_counter: 2,
+      likes_counter: 3,
+      author_id: @user.id
+    )
+    visit user_path(@user.id)
+  end
+
+  it 'shows the profile picture of the user' do
+    assert page.has_xpath?("//img[@src ='https://unsplash.com/photos/F_-0BxGuVvo'
+         and @alt='user profile']")
+  end
+
+  it 'shows the username' do
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'shows the bio' do
+    expect(page).to have_content(@user.bio)
+  end
+
+  it 'shows  number of posts' do
+    expect(page).to have_content(@user.posts_counter)
+  end
+
+  it 'shows last three posts of user' do
+    expect(page).to have_content(@post1.text)
+    expect(page).to have_content(@post2.text)
+    expect(page).to have_content(@post3.text)
+  end
```

Comment:
- [ ] [REQUIRED] This test fails: Tests that assert you can see the user's first 3 posts. However, line 52 of your code describes it as 'last three posts' even though you created new posts using the Post.new() method. Kindly fix these errors.
<img width="814" alt="Screenshot 2023-02-09 at 10 22 45" src="https://user-images.githubusercontent.com/58692590/217773663-79a5414f-4ea9-4dec-8007-ee675a4f5312.png">